src/LoginForm.jsx

```diff
@@ -185,7 +179,7 @@ class LoginForm extends React.Component {
           trySubscribe();
         }
       };
-      testUpdateLoop = setInterval(testConnectionUpdateLoop, 3000);
+      this.testUpdateLoop = setInterval(testConnectionUpdateLoop, 3000);
```

Comment:
`_cleanup()` cant reference a local variable, so I assume this should be `this.testUpdateLoop` and the variable with identical name inside this method should not exist.